website/src/_posts/2019-04-19-season-of-docs.md

```diff
@@ -0,0 +1,71 @@
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+The Apache Beam community is thrilled to announce its application to the first edition of  Season of Docs 2019! 
+
+<!--more-->
+
+<img src="{{ "/images/blog/SoD.png" | prepend: site.baseurl }}" alt="Season of Docs 2019 flyer" height="455" width="640" >
+
+[Season of Docs](https://developers.google.com/season-of-docs/) is a unique program that pairs technical writers with open source mentors to contribute to open source. This creates an opportunity to introduce the technical writer to an open source community and provide guidance while the writer works on a real world open source project. We, in the Apache Beam community, would love to take this chance and invite technical writers to collaborate with us, and help us improve our documentation in many ways.
+
+Apache Beam does have help from excellent technical writers, but the documentation needs of the project often exceed their bandwidth. This is why we are excited about this program.
+
+After discussing ideas in the community, we have been able to find mentors, and frame two ideas that we think would be a great fit for an incoming tech writer to tackle. We hope you will find this opportunity interesting - and if you do, please get in touch by emailing the Apache Beam mailing list at [dev@beam.apache.org](mailto:dev@beam.apache.org) (you will need to subscribe first by emailing to [dev-subscribe@beam.apache.org](mailto:dev-subscribe@beam.apache.org)).  
+
+The project ideas available in Apache Beam are described below. Please take a look and ask any questions that you may have. We will be very happy to help you get onboarded with the project.
+
+### Project ideas
+
+**Deployment of a Flink and Spark Clusters with Portable Beam**
+
+The Apache Beam vision has been to provide a framework for users to write and execute pipelines on the programming language of your choice, and the runner of your choice. As the reality of Beam has evolved towards this vision, the way in which Beam is run on top of runners such as Apache Spark and Apache Flink has changed.
+
+These changes are documented in the wiki and in design documents, and are accessible for Beam contributors; but they are not available in the user-facing documentation. This has been a barrier of adoption for other users of Beam.
+
+This project involves improving the Flink Runner page[1] to include strategies to deploy Beam on a few different environments: A Kubernetes cluster, a Google Cloud Dataproc cluster, and an AWS EMR cluster. There are other places in the documentation that should be updated in this regard[4][5].
```

Comment:
On the web, it looks more natural to link the text itself. I think in Markdown you can do this via:

```
... improving the [Flink Runner page] ...

[Flink Runner page](...)
```

But if that doesn't work with Jekyll, some other ideas. You can link these directly inline. But I do think the bullet list of links at the end is nice to read. So you could just have inline links and then a transition sentence, like "Here are the pages proposed for revision:"